specification/README.md

```diff
@@ -0,0 +1,13 @@
+# Ably Asset Tracking Specification
+
+## Overview
+
+This document is meant to be a formal, canonical specification describing the Ably Asset Tracking solution.
+
+## Error codes
+
+The list of Ably Asset Tracking specific error codes.
+
+| Code   | Name            | Description                                                                                                                                    |
```

Comment:
We can never predict how wide any of the columns will every need to be in markdown tables, so I always compact the heading rows down (more Git diff friendly as the table evolves)...

```suggestion
| Code | Name | Description |
```

Reply:
The auto-formatting did that for me and I had a feeling that it is not particularly good :sweat_smile: https://github.com/ably/ably-asset-tracking-common/pull/56/commits/bf751cb491e50ddbe68e18bd43b8da9a4b9a1c31